client/overview/connection-sucess-notice.tsx

```diff
@@ -3,34 +3,57 @@
  */
 import React from 'react';
 import { __ } from '@wordpress/i18n';
+import { Card, DropdownMenu } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import ConfettiImage from '../../assets/images/confetti.svg';
-import { Card } from '@wordpress/components';
 
 const ConnectionSuccessNotice: React.FC = () => {
+	const [ isDismissed, setIsDismissed ] = React.useState( false );
+
 	const {
 		accountStatus: {
 			progressiveOnboarding: { isComplete, isEnabled },
 		},
 	} = wcpaySettings;
 
-	return (
+	const DismissMenu = () => {
+		return (
+			<div className="wcpay-connection-success__dropdown">
```

Comment:
We can move the `className` prop to `DropdownMenu` and get rid of that extra div 🙂

Reply:
Nice one! https://github.com/Automattic/woocommerce-payments/pull/5992/commits/57bfd451b732f39a5d9c3d2b1d8617590aaa61aa